tests/lib/eslint/eslint.js

```diff
@@ -4885,6 +4885,71 @@ describe("ESLint", () => {
 
     });
 
+    describe("Use stats option", () => {
+
+        /**
+         * Check if the given number is a float.
+         * @param {number} n The number to check.
+         * @returns {boolean} `true` if the number is a float, `false` otherwise.
+         */
+        function isFloat(n) {
+            return Number(n) === n && n % 1 !== 0;
+        }
+
+        it("should report stats", async () => {
+            process.chdir(originalDir);
```

Comment:
Because you are doing this in both test cases, it would be better to move this into a `beforeEach`.